src/commands/aksCRUDDiagnostics/aksCRUDDiagnostics.ts

```diff
@@ -1,103 +1,74 @@
 import * as vscode from 'vscode';
 import * as k8s from 'vscode-kubernetes-tools-api';
 import { IActionContext } from "vscode-azureextensionui";
+import { getAksClusterTreeItem } from '../utils/clusters';
 import { getExtensionPath, longRunning }  from '../utils/host';
-import { AppLensARMResponse, getAppLensDetectorData } from '../utils/detectors';
-import * as fs from 'fs';
-import * as htmlhandlers from "handlebars";
-import path = require('path');
-import { htmlHandlerRegisterHelper } from '../utils/detectorhtmlhelpers';
+import { AppLensARMResponse, getDetectorInfo, getDetectorListData, getPortalUrl } from '../utils/detectors';
+import { failed } from '../utils/errorable';
 import AksClusterTreeItem from '../../tree/aksClusterTreeItem';
+import { createWebView, getRenderedContent, getResourceUri, HtmlHelper } from '../utils/webviews';
 
 export default async function aksCRUDDiagnostics(
-    context: IActionContext,
+    _context: IActionContext,
```

Comment:
💡 Thank you for this, can we please add more context for this change and benefits.

Reply:
It was something I changed to keep the linter in vscode happy (otherwise it complains that `context` is unused). This prompted me to go searching for an official style guide that says unused parameters should be prefixed by `_`.

[This](https://github.com/microsoft/TypeScript/issues/9458) is the closest thing to that I was able to find. It's apparently a little controversial (some people like underscore prefixes for parameters that _are_ used), and an alternative would be to disable the linter rule. I don't think the convention introduces any inconsistencies with code in this project, so I think it's fine here.